service/src/test/java/greencity/mapping/events/EventDateLocationDtoMapperTest.java

```diff
@@ -0,0 +1,35 @@
+package greencity.mapping.events;
+
+import greencity.dto.event.EventDateLocationDto;
+import greencity.entity.event.EventDateLocation;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static greencity.ModelUtils.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class EventDateLocationDtoMapperTest {
+    @Spy
```

Comment:
Why do you use `Spy` here?

Reply:
I use @Spy here because I need the real behavior of the convert method of CoordinatesDtoMapper class which is used in EventDateLocationDtoMapper, in case I use @Mock I don't get this convert and get null